games-roguelike/dwarf-fortress/dwarf-fortress-0.47.05.ebuild

```diff
@@ -73,12 +89,11 @@ src_install() {
 pkg_postinst() {
 	elog "System-wide Dwarf Fortress has been installed to ${gamesdir}. This is"
 	elog "symlinked to ~/.dwarf-fortress when dwarf-fortress is run."
-	elog "For more information on what exactly is replaced, see /usr/bin/dwarf-fortress."
-	elog "Note: This means that the primary entry point is /usr/bin/dwarf-fortress."
+	elog "For more information on what exactly is replaced, see ${EPREFIX}/usr/bin/dwarf-fortress."
+	elog "Note: This means that the primary entry point is ${EPREFIX}/usr/bin/dwarf-fortress."
 	elog "Do not run ${gamesdir}/libs/Dwarf_Fortress."
```

Comment:
Is it necessary to show this wall of elog every time? Is once using REPLACING_VERSIONS enough? Or maybe go with [readme.gentoo-r1](https://devmanual.gentoo.org/eclass-reference/readme.gentoo-r1.eclass/index.html)?